config/routes.rb

```diff
@@ -10,7 +10,10 @@
   delete '/shelters/:id', to: 'shelters#destroy'
   get '/pets', to: 'pets#index'
   get '/shelters/:id/pets', to: 'shelters#show_pets'
-  get 'pets/:id', to: 'pets#show'
+  get '/favorites', to: 'pets#show_favorite'
+  delete '/pets/:id/fav', to: 'pets#remove_favorite'
```

Comment:
Good catch on this as well! Makes sense to add /fav to the end to make the route clearer!